scripts/demangle.py

```diff
@@ -2,23 +2,38 @@
 import json
 import glob
 import re
+import docker
 import ctypes
+import cpp_demangle
 
-json_path = '../dump/augeas'
-proj_path = '../oss-fuzz/projects/augeas'
-file_names = [f for f in os.listdir(json_path) if f.endswith('.json') and os.path.isfile(json_path+'/'+f)]
-print('Looking for json file: ',file_names)
-def extract_func_code(func_name, file_path):
-    with open(file_path, 'r') as file:
-        code_content = file.read()
+proj_name = 'tmux'
+json_path = '../dump/'+proj_name
```

Comment:
there is a env variable for the workdir of this project in `common.py` to import, so use
```py
json_path = os.path.join(OSSFUZZ_SCRIPTS_HOME, "dump", proj_name)
```

Reply:
How can I get the OSSFUZZ_SCRIPTS_HOME from common.py? I just define it in my code.

Reply:
```py
from common import OSSFUZZ_SCRIPTS_HOME
```